tests/integration/test_logcollector/test_macos/data/wazuh_macos_file_status_predicate.yaml

```diff
@@ -0,0 +1,30 @@
+- tags:
+    - test_macos_file_status_predicate
+  apply_to_modules:
+    - test_macos_file_status_predicate
+  sections:
+    - section: localfile
+      attributes:
+        - name: 'localfile_macos_block'
+      elements:
+        - location:
+            value: 'macos'
+        - log_format:
+            value: 'macos'
+        - query:
+            value: 'message =x= "tresting merssage"'
```

Comment:
Should it be `"testing message"`?

Reply:
no, it is just an "erroneous message" that a user might write when creating the predicate